lib/internal/Magento/Framework/App/Test/Unit/MaintenanceModeTest.php

```diff
@@ -202,18 +202,251 @@ public function testSetSingleAddresses()
 
         $this->flagDir->method('readFile')
             ->with(MaintenanceMode::IP_FILENAME)
-            ->willReturn('address1');
+            ->willReturn('198.51.100.1/32');
```

Comment:
I believe we still should support regular IP address, just 198.51.100.1.

Could you add back such use case?

Reply:
I've added a test to ensure that raw IP addresses (old/current file format) still work.